### PR TITLE
Components

### PR DESCRIPTION
**Pull request by initseis**
In this branch I:

- Created a component in a different JS file
- Imported the component in the App.js
- Added style to created component
- Updated Linters

image

### CODE REVIEW

**Comment by initseis** [comment]
Hi, reviewer. I don't know why the linters are not appearing.

**Comment by DeliceLydia** [comment]
# Status: Invalid

Hello @sevinchek,

I can see you worked hard but I can't review your project because no linter checks
available now please follow this [link](https://github.com/microverseinc/linters-config/tree/master/react-redux) step by step to configure it well and then submit another CRR

Thank you!

**Comment by initseis** [comment]
Hi, @DeliceLydia. I already resolved it.

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,129 @@
...
-  color: #61dafb;
+#ac {
+  grid-area: ac;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+#percentage {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,129 @@
...
+  grid-area: percentage;
+}
+
+#division {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 3]
```diff
@@ -1,39 +1,129 @@
...
+  grid-area: division;
+}
+
+#seven {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 4]
```diff
@@ -1,39 +1,129 @@
...
+  grid-area: seven;
+}
+
+#eight {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 5]
```diff
@@ -1,39 +1,129 @@
...
+  grid-area: eight;
+}
+
+#nine {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 6]
```diff
@@ -1,39 +1,129 @@
...
+  grid-area: nine;
+}
+
+#by {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 7]
```diff
@@ -1,39 +1,129 @@
...
+
+#zero {
+  grid-area: zero;
+}
+
+#dot {
+  grid-area: dot;
+}
+
+#equal {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 8]
```diff
@@ -1,39 +1,129 @@
...
+
+#plus {
+  grid-area: plus;
+}
+
+#zero {
+  grid-area: zero;
+}
+
+#dot {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.css [hunk 9]
```diff
@@ -1,39 +1,129 @@
...
+
+#three {
+  grid-area: three;
+}
+
+#plus {
+  grid-area: plus;
+}
+
+#zero {
```
Comment (arslanbisharat) [inline src/App.css]: - [ ] Try to avoid using ID aa a selector. You can use classes instead.

File: src/App.js [hunk 10]
```diff
@@ -1,25 +1,8 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Calculator />;
 }
 
 export default App;
```
Comment (arslanbisharat) [inline src/App.js:16]: - [ ] App js should have a class based component. Please work on it.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js [hunk 11]
```diff
@@ -1,25 +1,16 @@
...
-  );
+class App extends React.Component {
+  render() {
+    return <Calculator />;
+  }
 }
 
+// function App() {
+//   return <Calculator />;
+// }
```
Comment (Davidosky007) [inline src/App.js:14]: ## [OPTIONAL]
* I strongly suggest you remove commented code it makes your codebase bulky and unprofessional

**Review by Davidosky007 — APPROVED** [review]
## Hi @sevinchek 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by initseis** [comment]
Thanks, @Davidosky007.

**Pull request merged**